subsys/net/lib/azure_iot_hub/Kconfig

```diff
@@ -99,6 +99,14 @@ config AZURE_IOT_HUB_PROPERTY_BAG_MAX_COUNT
 	  Increasing this value will increase stack usage when parsing topics,
 	  and vice versa if decreasing it.
 
+config AZURE_IOT_HUB_TOPIC_PROPERTY_BAG_PREFIX
+	bool "Include the <?> prefix before the property bag key/value"
```

Comment:
I'd change `<?>` to `'?'`, and let the example string below use `?` only

Reply:
Done.